_posts/2018-07-17-its-a-lot-more-snappy.md

```diff
@@ -131,13 +127,13 @@ Now that we had implemented this pattern, we noticed that the initial download o
 
 It was at this point our fearless tech lead, William Sia(TM) proposed using a web worker for that exact purpose. This was new and very exciting. Of course we wanted to use a web worker!
 
-I found a module called <a href="https://www.npmjs.com/package/worker-loader">worker-loader</a> which seamlessly integrated with our current build and packaging eco-system. In fact, worker-loader almost made it inconceivable that we waited such a long time to implement our first web worker. We didn’t even need to add any additional config to our web pack config files.
+I found a module called [worker-loader](https://www.npmjs.com/package/worker-loader) which seamlessly integrated with our current build and packaging eco-system. In fact, worker-loader almost made it inconceivable that we waited such a long time to implement our first web worker. We didn’t even need to add any additional config to our web pack config files.
```

Comment:
`found`

Choose or choose not, there is no found...

Reply:
This is a good paragraph though - just how easy web workers are.  Include the benefit it gives here and 💥